packages/@webex/plugin-cc/src/services/AgentService.ts

```diff
@@ -56,4 +65,28 @@ export default class AgentService {
       return Promise.reject(error);
     }
   }
+
+  public async stationLogout(options: {logoutReason: string}): Promise<LogoutSuccess> {
```

Comment:
The return value should be generic right, either the promise resolves or rejects indicating success or failure

Reply:
Done